src/pages/task/taskTabs/buildBaronAndAnnotations/BuildBaron.test.tsx

```diff
@@ -304,14 +304,14 @@ const ticketsTableMocks = [
       query: MOVE_ANNOTATION,
       variables: { taskId, execution, apiIssue, isIssue: true },
     },
-    result: { success: true },
+    result: { data: { moveAnnotationIssue: true } },
   },
   {
     request: {
       query: REMOVE_ANNOTATION,
       variables: { taskId, execution, apiIssue, isIssue: true },
     },
-    result: { success: true },
+    result: { data: { removeAnnotationIssue: true } },
```

Comment:
I got this wrong in my other PR that I merged I am sorry _(┐「ε:)_ 

(It doesn't fail because the return value is not actually used, it just needs the mock definitions)